kafka-streams-framework/src/main/java/org/hypertrace/core/kafkastreams/framework/KafkaStreamsApp.java

```diff
@@ -67,6 +75,13 @@ public abstract class KafkaStreamsApp extends PlatformService {
 
   public KafkaStreamsApp(ConfigClient configClient) {
     super(configClient);
+    this.grpcChannelRegistry =
```

Comment:
Do all the kstream apps need this? Shouldn't this be an extension of GrpcKstreamApp or something?

Reply:
> Do all the kstream apps need this?

Yes. Almost all of them use it for their application logic anyway, but all of them will need to use grpc to use the weighted group partitioner.